include/libsemigroups/paths.hpp

```diff
@@ -848,6 +494,7 @@ namespace libsemigroups {
   //! the number of paths originating at the given source node and ending at
   //! the given target node with length in the range \f$[min, max)\f$.
   //!
+  //! \param wg the WordGraph
```

Comment:
```suggestion
  //! \param wg the WordGraph.
```
(and the same for the params below)